### DISCUSSION TITLE
Welcome to UCBerkeley-EE120-Lab-Solutions Discussions!

### DISCUSSION THREAD

**Discussion by cy-Yin** [cy-Yin/UCBerkeley-EE120-Lab-Solutions#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gaofrank1986** [comment]
hi，it seems that the course site need  calnet account to see. how do you solve this?

**Reply by cy-Yin** [reply]
I finished this course one year ago. At that time the website did not require a CalNet account. 
I am sorry for the current situation.
